less/plugins/adapt-contrib-hotgraphic/hotgraphic.less

```diff
@@ -26,4 +26,39 @@
     text-align: center;
   }
   // --------------------------------------------------
+  // _useGraphicsAsPins config
+  // --------------------------------------------------
+  // _use
+  &__tile-item {
+    position: relative;
+    z-index: 4;
+
+    .hotgraphic__tile {
```

Comment:
Suggest moving the nested styles outside of the tile item element. can also make use of `&__` so don't have to repeat the word `hotgraphic`

```
&__tile:before {
  content: '';
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  background-color: fadeout(@item-color, 100%);
  .transition(background-color @duration ease-in);
}
```